crates/hc_sandbox/src/calls.rs

```diff
@@ -1,9 +1,9 @@
-//! Helpers for making [`AdminRequest`]s to the admin API.
+//! Helpers for making [`holochain_conductor_api::AdminRequest`]s to the admin API.
 //!
 //! This module is designed for use in a CLI so it is more simplified
-//! than calling the [`CmdRunner`] directly.
-//! For simple calls like [`AdminRequest::ListDnas`] this is probably easier
-//! but if you want more control use [`CmdRunner::command`].
+//! than calling the [`AdminWebsocket`] directly.
+//! For simple calls like [`AdminWebsocket::list_dnas`] this is probably easier
+//! but if you want more control use [`AdminWebsocket::send`].
```

Comment:
`send` isn't public and shouldn't be - I think probably just drop the tail end of this doc comment?